tasks/sealights/sealights-get-refs/0.1/sealights-get-refs.yaml

```diff
@@ -63,13 +63,22 @@ spec:
             | .results[] | select(.name == "SOURCE_ARTIFACT") | .value' \
             cosign_metadata.json)"
 
-        SL_CONTAINER_IMAGE="$(jq -r \
+        SL_IMAGE_URL="$(jq -r \
             --arg sl_source_artifact "$SL_SOURCE_ARTIFACT" \
             '.payload | @base64d | fromjson | .predicate.buildConfig.tasks[] |
             select(.invocation.parameters."SOURCE_ARTIFACT" == $sl_source_artifact)
-            | .results[] | select(.name == "IMAGE_REF") | .value' \
+            | .results[] | select(.name == "IMAGE_URL") | .value' \
```

Comment:
ok so the uncertainty about which image the `IMAGE_REF` points to comes from the fact that the results are extracted from more than one Task, right?

so what if we'd remove this ambiguity by filtering only the Task that references the `buildah-oci-ta` task?
```
jq -r \
--arg sl_source_artifact "$SL_SOURCE_ARTIFACT" \
'.payload | @base64d | fromjson | .predicate.buildConfig.tasks[] |
select(.invocation.parameters."SOURCE_ARTIFACT" == $sl_source_artifact) | \
select(.ref.params[].value == "buildah-oci-ta") | .results[] | select(.name == "IMAGE_REF") | .value' cosign_metadata.json
```

Reply:
fixed! Thank you @psturc !